src/render.rs

```diff
@@ -133,7 +144,13 @@ impl<'a> MarkdownRenderer<'a> {
                 let offset = new_url.len() - 5;
                 new_url.drain(offset..offset + 4);
             }
-            new_url += "blob/master";
+            // Assumes GitHub's URL scheme. GitHub renders text and markdown
```

Comment:
It looks like removing the apostrophe here fixes things.  It is also strange that these are reported as errors, even though we invoke with `cargo fmt -- --write-mode=diff` which should provide a diff.

Trying with rustfmt on beta also seems to work fine, so I don't think we need to report this upstream as it seems to be resolved there.

Maybe just reword a bit so that it can land now?

Reply:
OK, thanks